### PR TITLE
Add My User Account Info Page

### PR DESCRIPTION
**Pull request by ji-mmyliu**
## Notion ticket link

[User Account Page](https://www.notion.so/uwblueprintexecs/User-Account-Page-2250e1cb67c0428cb33849944e3bb610)

## Implementation description
* Added a button in the navbar for "My Account" which takes you to the account info page at `/account`
* The account info page shows the info for your account taken from `authenticatedUser`.
* A private route was used to ensure that a user is logged in.

## Steps to test
1. Press the "My Account" button in the navbar. 
2. See if your account info appears on the screen
3. Try going to the page without being logged in and see if you are redirected to the login page.

## What should reviewers focus on?
* Functionality and style of code.

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by carolynzhang18 — APPROVED** [review]
NICE!! 🤩🤩

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ import HooksDemo from "./components/pages/HooksDemo";
 import { AuthenticatedUser } from "./types/AuthTypes";
 import authAPIClient from "./APIClients/AuthAPIClient";
 import * as Routes from "./constants/Routes";
+import MyAccount from "./components/pages/MyAccount";
```
[the remark this answers is not in the document]
Comment (ji-mmyliu) [reply frontend/src/App.tsx]: You got it :+1:

**Comment by 98ZhaoJeffrey** [comment]
fire

**Review by 98ZhaoJeffrey — APPROVED** [review]
fire

**Pull request merged**